transaction/core/src/validation/validate.rs

```diff
@@ -920,6 +945,46 @@ mod tests {
         }
     }
 
+    #[test]
+    /// Should reject a transaction with unsorted outputs.
+    fn test_validate_outputs_are_sorted() {
+        for block_version in BlockVersion::iterator() {
```

Comment:
Let's extend this test (or add another) to verify that `validate` rejects transactions with unsorted outputs if `BlockVersion.validate_transaction_outputs_sorted`, and accepts them otherwise.

Reply:
can you help me with this. I checked the other test we have and could not find a direct call to `validate` itself. Mainly I don't know how to correctly create all the needed parameters besides `tx and block_version`. Thx!

Reply:
Ah, I see there are no other tests that call the top-level `validate()`. This can be a follow-up, then.